src/reset-password/ResetPasswordPage.jsx

```diff
@@ -65,7 +67,9 @@ const ResetPasswordPage = (props) => {
       props.resetPassword(formPayload, props.token, params);
     }
   };
-
+  if (props.errorCode === INTERNAL_SERVER_ERROR) {
+    return (<APIFailureMessage />);
```

Comment:
Can we use the same approach used at [L88](https://github.com/edx/frontend-app-authn/pull/84/files#diff-62ac94b242d30a85a60109539e352737262794426cba6505ec68c8459004e65bR88)?